### ISSUE TITLE
(1.2) `Flipper::TestHelp` initializer raises an error if enabled

### ISSUE THREAD

**Issue by boardfish** [flippercloud/flipper#818]
This is with Ruby 3.3, Rails 7.1.2. The solution right now is to set `config.flipper.test_help = false` in `config/application.rb` so that the affected code isn't run.

This might be fixed (in a way) on `main` since [the conditional preventing this line from being run has been changed](https://github.com/flippercloud/flipper/blob/main/lib/flipper/test_help.rb#L19-L20), but [in 1.2.0 it would allow this](https://github.com/flippercloud/flipper/blob/v1.2.0/lib/flipper/test_help.rb#L19-L20).

```rb
Run bin/rails db:create db:schema:load
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
bin/rails aborted!
NoMethodError: undefined method `configure' for module RSpec (NoMethodError)

  RSpec.configure do |config|
       ^^^^^^^^^^
my_rails_app/vendor/bundle/ruby/3.3.0/gems/flipper-1.2.0/lib/flipper/test_help.rb:20:in `<main>'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/bootsnap-1.17.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/zeitwerk-2.6.12/lib/zeitwerk/kernel.rb:38:in `require'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/flipper-1.2.0/lib/flipper/engine.rb:91:in `block in <class:Engine>'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/railties-7.1.2/lib/rails/initializable.rb:32:in `instance_exec'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/railties-7.1.2/lib/rails/initializable.rb:32:in `run'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/railties-7.1.2/lib/rails/initializable.rb:61:in `block in run_initializers'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/railties-7.1.2/lib/rails/initializable.rb:60:in `run_initializers'
my_rails_app/vendor/bundle/ruby/3.3.0/gems/railties-7.1.2/lib/rails/application.rb:423:in `initialize!'
my_rails_app/config/environment.rb:5:in `<main>'
```

**Comment by boardfish** [comment]
I'm not 100% sure the current fix (to check that `RSpec` responds to `configure`) is appropriate, since I have `rspec-rails` installed in this repo and I figure `TestHelp` would want to configure these hooks. Perhaps there's something to do with load order going on here.

**Comment by bkeepers** [comment]
Already fixed by #816, which should get released today.

**Comment by bkeepers** [comment]
1.2.1 is out now.
